### ISSUE TITLE
Using Vue OverlayScrollbarsComponent.osInstance() got null

### ISSUE THREAD

**Issue by HaceraI** [KingSora/OverlayScrollbars#293]
This is the first time that I've applied OverlayScrollbars to my Vue project, and now I've got a little problem

The version I use is:
  overlayscrollbars: ‘1.13.1’
  overlayscrollbars-vue: '0.2.2'

Before using it, I looked up the official example: 
https://github.com/KingSora/OverlayScrollbars/blob/master/packages/overlayscrollbars-vue/example/src/App.vue

I want to change the scroll of overlay scrollbars. First, I call osinstance () to get the instance, but when I use this method, it returns a null
Can you take a look for me?
![image](https://user-images.githubusercontent.com/48341368/103348098-fe4dd280-4ad3-11eb-9666-ccf3fc4d0bf9.png)
![image](https://user-images.githubusercontent.com/48341368/103348160-29382680-4ad4-11eb-8102-9819650b6b69.png)
When osinstance () returns null
![image](https://user-images.githubusercontent.com/48341368/103348466-0d815000-4ad5-11eb-8634-f3cb2051268d.png)
![image](https://user-images.githubusercontent.com/48341368/103348418-e62a8300-4ad4-11eb-99a1-253a35c14756.png)

If there is anything wrong with me, I hope you can point it out. Thank you very much for your help

Tips: English is not my native language; please excuse typing errors.

**Comment by KingSora** [comment]
Good day @HaceraI!

Please create a small example of your problem in [CodeSandbox](https://codesandbox.io/) or [StackBlitz](https://stackblitz.com/). I can't spot anything wrong in your provided screenshots.

**Comment by HaceraI** [comment]
@KingSora of course.
This is CodeSandbox: https://codesandbox.io/s/nifty-framework-4ubb8?file=/src/App.vue:2248-2293

**Comment by HaceraI** [comment]
@KingSora of course
This is the CodeSandbox link: https://codesandbox.io/s/nifty-framework-4ubb8?file=/src/App.vue:2248-2293

**Comment by KingSora** [comment]
Thanks!

I got the issue, but I'm not sure why it happens yet. You see, the `osInstance` is available after the `mounted` lifecycle method, but for some reason I don't know yet the `onContentSizeChanged` callback is called before that. Thats also why your `if` condition is fixing it.. I'll look deeper into it as soon as I have a bit more time.

**Comment by HaceraI** [comment]
I hope the information I provide will be helpful to you:

 I got it OverlayScrollbars.js The final call to `onContentSizeChanged` is found in. This is a link to the related code: 
https://github.com/KingSora/OverlayScrollbars/blob/master/js/OverlayScrollbars.js#L6482-L6486

After the `_initialized` is set to true, the `dispatchCallback` is called. 
I wonder if every `callbacks` will be called once after initialization?

To sum up, I have tried some successful methods:
 1: Using lazy loading  (`const OverlayScrollbarsComponent = () => import("overlayscrollbars-vue");`)
 2: Add verification before use

Finally, I uploaded the code to CodeSandbox, which you can visit and view at any time

I wish you happy every day.

**Comment by KingSora** [comment]
`overlayscrollbars` `v2` released today and with it a new version of `overlayscrollbars-vue` where this issue should be fixed
